### PR TITLE
link to pages from nav menu

### PR DESCRIPTION
**Pull request by LeeRenJie**
depends on #80

### CODE REVIEW

File: message-me/app/views/layouts/_navigation.html.erb [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
+<%= link_to 'Messages', root_path, class: "item" %>
 <div class="right menu">
   <div class="ui dropdown item">
     Account
     <i class="dropdown icon"></i>
     <div class="menu">
-      <a href="" class="item">Log out</a>
-      <a href="" class="item">Log in</a>
-      <a href="" class="item">Sign Up</a>
+      <%= link_to 'Log out', login_path, class: "item" %>
```
Comment (tlkiong) [inline message-me/app/views/layouts/_navigation.html.erb:8]: better to not use any link to if it don't exist yet so that you don't forget

**Review by tlkiong — CHANGES_REQUESTED** [review]

File: message-me/app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -1 +1,38 @@
-<h1>this is new.html.erb for sessions</h1>
+<h1 class="ui header center aligned">Welcome to MessageMe - a complete Chat App</h1>
+<h4 class="ui center aligned medium icon header">
+  <i class="orange circular power off icon"></i>
+  Log in to continue
```
Comment (tlkiong) [inline message-me/app/views/sessions/new.html.erb:4]: ???

File: message-me/app/views/sessions/new.html.erb [hunk 3]
```diff
@@ -1 +1,38 @@
...
+</h4>
+
+<div class="ui placeholder segment">
+  <div class="ui two column very relaxed stackable grid">
+    <div class="column">
+      <div class="ui form">
```
Comment (tlkiong) [inline message-me/app/views/sessions/new.html.erb:10]: form tag?

**Review by tlkiong — CHANGES_REQUESTED** [review]
please double triple check ya

File: message-me/app/views/sessions/new.html.erb (hunk 2 above)
Comment (LeeRenJie) [reply message-me/app/views/sessions/new.html.erb]: forgot merge 😅

**Review by tlkiong — APPROVED** [review]

**Pull request merged**